planner/plan.go

```diff
@@ -42,8 +42,11 @@ type SharedPlan struct {
 }
 
 type Stage struct {
-	Command string `cue:"string" json:"command"`
-	Image   string `json:"-"`
+	Command    string   `cue:"string" json:"command"`
+	InputFiles []string `cue:"[...string]" json:"input_files,omitempty"`
+	Scripts    []string `cue:"[...string]" json:"scripts,omitempty"`
```

Comment:
Two thoughts:
1. Could we just make these part of command for now to avoid introducing a new field?
i.e. your command can be:
```
script1 && script2 && do-something
```
We could even allow command to be an in-line script (see https://docs.docker.com/engine/reference/builder/#example-creating-inline-files)

Which would let you write a script without all the `&&`:
```
StartCommand: heredoc.Doc(`
    step 1
    step 2
    step 3
    command
`)
```

2. If we really do need a field like this, I'd prefer to call it something like `preCommandHooks`

Reply:
This is mostly for the `Entrypoint` field in Dockerfile - do we want the `Entrypoint` to be `script1 && script2 && startCommand`? Or should it be `Run script1 && script2` and `Entrypoint startCommand`?